base/distributed/managers.jl

```diff
@@ -455,10 +455,10 @@ end
 const client_port = Ref{Cushort}(0)
 
 function socket_reuse_port()
-    @static if is_linux() || is_apple()
+    if Bool(ccall(:jl_has_so_reuseport, Int32, ()))
```

Comment:
Right, I was talking more specifically about the return type of `jl_has_so_reuseport`, which will only ever be 0 or 1, so I figured that one was safe to convert to `Bool`.